app/views/shared/_recaptcha.html.slim

```diff
@@ -1,4 +1,6 @@
 - if FeatureManagement.recaptcha_enabled?(session, true)
+  - SecureHeaders.append_content_security_policy_directives(\
+    request, style_src: ['\'unsafe-inline\''])
```

Comment:
We do not want to use unsafe-inline for anything, as it opens up the site for cross site styling attacks: https://stackoverflow.com/questions/30653698/csp-style-src-unsafe-inline-is-it-worth-it

If we want to use inline styles, we must whitelist them via a hash or nonce.

Reply:
Unfortunately dynamic style is the only thing that won't work with hash or nonce.  The html google inserts doesn't even have an id or class.